src/python/ripe_demo/static/js/main.js

```diff
@@ -10,8 +10,12 @@ window.onload = function() {
     var parts = [];
     var partsMap = {};
     var index = 0;
+    var frames = {

```

Comment:
this mustn't be hardcode.

use the `/api/brands/<str:brand>/models/<str:model>/config` endpoint

Reply:
The available frames of a product can't be inferred with information currently provided:

```
"frames": 24,
"faces": ["side", "top"]
```